lldb/source/Core/Progress.cpp

```diff
@@ -16,7 +16,7 @@ using namespace lldb_private;
 
 std::atomic<uint64_t> Progress::g_id(0);
 
-Progress::Progress(std::string title, uint64_t total,
+Progress::Progress(std::string title, bool is_discrete, uint64_t total,
```

Comment:
You added this parameter but it doesn't look like it's used anywhere in `Progress`? Was this supposed to go to `m_is_discrete`?

Reply:
Yes it was, left it out by mistake.